src/org/zaproxy/zap/extension/pscanrulesBeta/resources/help/contents/pscanbeta.html

```diff
@@ -11,7 +11,23 @@
 The following beta quality passive scan rules are included in this add-on:
 
 <H2>Charset Mismatch</H2>
-This check identifies responses where the HTTP Content-Type header declares a charset different from the charset defined by the body of the HTML or XML. When there's a charset mismatch between the HTTP header and content body Web browsers can be forced into an undesirable content-sniffing mode to determine the content's correct character set.
+<p>This check identifies responses where the HTTP Content-Type header declares a charset different from the charset defined by the body of the HTML or XML. When there's a charset mismatch between the HTTP header and content body Web browsers can be forced into an undesirable content-sniffing mode to determine the content's correct character set.</p>
+<p>The scanner handles various conditions depending on the Threshold set for the scanner, as follows:</p>
+<li>All Thresholds:
```

Comment:
These "li" elements ("All..." and "Low...") should also be in an "ul" element (as they were before).